### PR TITLE
Make document_id the only ID for reimbursements

### PR DESCRIPTION
**Pull request by cuducos**
When we first conceived Jarbas we weren't aware that `document_id` was, in the CEAP dataset, an unique id. On the contrary: we thought a combination of `applicant_id`, `year` and `document_id` would give us an unique identifier for each reimbursement.

Now that we know we can trust `document_id` as an unique identifier for reimbursement this PR:

* Simplifies a lot of the backend code (no more `unique_together`, no more custom `MultipleFieldLookupMixin` classes etc.)
* Simplifies a lot the API (no more `/api/reimbursement/<year>/<applicant_id>/<document_id>/` just `/api/reimbursement/<document_id>/`

### CODE REVIEW

File: jarbas/core/management/commands/irregularities.py [hunk 1]
```diff
@@ -45,37 +44,42 @@ def irregularities(self):
 
     def serialize(self, row):
         """
-        Read the dict generated by DictReader and return a tuple with two
-        dictionaries, one to be used as filter to locate the reimbursement,
-        another one with data from the irregularities.
+        Read the dict generated by DictReader and returns another dict with
```
Comment (jtemporal) [inline jarbas/core/management/commands/irregularities.py]: 💅 Either there is one s missing on Read or one left on returns not quite sure

**Comment by cuducos** [comment]
Typos fixed ; )

**Comment by jtemporal** [comment]
#164 was accepted which generated a few conflicts... could you check them out @cuducos ?

File: README.md [hunk 2]
```diff
@@ -25,21 +25,21 @@ Jarbas is in charge of making data from [CEAP](https://github.com/datasciencebr/
...
-Each `Reimbursement` object is a reimbursement claim made by a congressperson. Each reimbursement isidentified by an unique combination of `year`, `applicant_id` and `document_id`.
+Each `Reimbursement` object is a reimbursement claimed by a congressperson and identified publicly by its `document_id`.
 
 #### Retrieving a specific reimbursement
 
-##### `GET /api/reimbursement/<year>/<applicant_id>/<document_id>/`
+##### `GET /api/reimbursement/<document_id>/`
 
-Details from a specific reimbursement. If `receipt_url` wasn't fecthed yet, the server **won't** try to fetche it.
+Details from a specific reimbursement. If `receipt_url` wasn't fetched yet, the server **won't** try to fetche it automatically.
```
Comment (jtemporal) [inline README.md]: lol you corrected a typo I did not see, `try to fetche it` -> `try to fetch it` there is an extra `e` there =P

**Comment by jtemporal** [comment]
Apart from the typos and conflicts LGTM o/

This was a great one @cuducos ;)

**Pull request merged**